docs/en/api/wrapper/visible.md

```diff
@@ -0,0 +1,21 @@
+# visible()
+
+Assert `Wrapper` or `WrapperArray` is visible.
+
+Returns false if an ancestor element has `display: none` or `visibility: hidden` style.
+
```

Comment:
We should add that this is useful for components hidden by v-show

Reply:
Done